.env.sample

```diff
@@ -7,10 +7,12 @@
 HARBOR_USER='admin'
 HARBOR_PASSWORD=''
 HARBOR_BASE_URL='http://127.0.0.1:8083/api/v2.0'
+HARBOR_BASE_REPO_URL='harbor.dev.eks.otomi.cloud'
 OIDC_CLIENT_SECRET=''
 OIDC_ENDPOINT='https://keycloak.dev.gke.otomi.cloud'
-OIDC_VERIFY_CERT='true' # set to false when staging certs are used on the cluster
-TEAM_NAMES='["team-otomi","team-chai","team-dev","team-demo"]'
+OIDC_VERIFY_CERT='true'
+#TEAM_NAMES='["team-otomi","team-chai","team-dev","team-demo"]'
```

Comment:
Why remove 2 teams & leave the previous value in place as a comment?

Reply:
I was testing my code and only team-demo and team-otomi namespaces were available in the dev cluster.

Reply:
But this is a sample file, so I'm not sure what teams are or are not available.